pkg/kube/util/names/docker_source.go

```diff
@@ -0,0 +1,27 @@
+package names
+
+import "errors"
+
+// GetDockerSourceName returns the name of the docker image
+// More info:
+//   * https://kubernetes.io/docs/concepts/containers/images
+//   * <hub-user>/<repo-name>[:<tag>]
+//   * ACCOUNT.dkr.ecr.REGION.amazonaws.com/imagename:tag
+//
+// prefix: [<host>[:<port>]/][<repo>/]
+// repo: <name>
+// tag :[:<tag>]
+func GetDockerSourceName(prefix, repo, tag string) (string, error) {
+	source := ""
+	if repo == "" {
+		return "", errors.New("repository is mandatory")
+	}
+	if prefix != "" {
+		source = prefix + "/"
```

Comment:
The docs above say that `prefix` should already end with a `/`. Maybe use `path.Join()` instead to make it robust?

Reply:
:+1: